CONTRIBUTING.md

```diff
@@ -1,34 +1,32 @@
 # How to contribute
 
-Big Calendar loves to welcome your contributions. There are several ways to help out:
+Big Calendar warmly welcomes your contributions. There are several ways to help out:
 
 - Create an [issue](https://github.com/jquense/react-big-calendar/issues) on GitHub, if you have found a bug
-- Write test cases or provide examples for open bug issues. You can fork our [Issue Template](https://codesandbox.io/s/react-big-calendar-example-v9wdyd) in codesandbox to get start.
+- Write test cases or provide examples for open bug issues. You can fork our [Issue Template](https://codesandbox.io/s/react-big-calendar-example-v9wdyd) in CodeSandbox.
 - Write patches for open bug/feature issues
 
-There are a few guidelines that we need contributors to follow so that we have a
-chance of keeping on top of things.
+To ensure efficiency and organization, we ask all contributors to follow a few guidelines.
 
 ## Getting Started
 
-- Make sure you have a [GitHub account](https://github.com/signup/free).
-- Submit an [issue](https://github.com/jquense/react-big-calendar/issues), assuming one does not already exist.
-  - Clearly describe the issue including steps to reproduce when it is a bug.
-  - Make sure you fill in the earliest version that you know has the issue.
-- Fork the repository on GitHub.
+- **Create a GitHub Account**: Sign up at [GitHub](https://github.com/signup/free) if you don't already have an account.
+- **Report Issues**: Before submitting a new issue, check if it already exists [here](https://github.com/jquense/react-big-calendar/issues).
```

Comment:
Rather than putting the link on 'here', put the link around 'check if it already exists'